ci-operator/step-registry/aws/provision/disconnectedusers/aws-provision-disconnectedusers-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: aws-provision-disconnectedusers
+  from: upi-installer
+  cli: latest
+  grace_period: 10m
+  commands: aws-provision-disconnectedusers-commands.sh
```

Comment:
It seems all the step does is creating the required policies/users for the installer. If so, why not remove "disconnected" and call it something "cco-manual-users"?